src/WinWrapper.cpp

```diff
@@ -331,6 +335,15 @@ int main(int argc, char** argv)
     // DXGI Factory
     ThrowIfFailed(CreateDXGIFactory2(0, IID_PPV_ARGS(&gDXGIFactory)));
 
+    if (gSettings.allowTearing)
+    {
+        if (gSettings.vsync)
+        {
+            std::cout << "Tearing cannot be used when vsync is on. Disabling it!" << std::endl;
```

Comment:
Can you tweak this to make it clear you're disabling allowTearing and not disabling vsync?

Reply:
done